lib/reducers/interface.js

```diff
@@ -0,0 +1,25 @@
+const { UPDATE_TEXT_FIELD, TEXT_STORE_KEY } = require('../constants');
+
+const initialState = {
+  [TEXT_STORE_KEY]: {},
+};
+
+function reduceInterface(state = initialState, action) {
```

Comment:
Not sure if you have any feelings on this, but if we're going to keep the naming consistent with the main bpanel project, the reducer functions are actually called like "interfaceStore"